support/ruby/consolr/bin/consolr

```diff
@@ -17,6 +17,7 @@ opt_parser = OptionParser.new do |opt|
   opt.on('-k', '--kick',            'kick if someone is hogging the console')  { options[:kick] = true }
   opt.on('-l', '--log LOG',         'System Event Log (SEL) [list|clear]')     { |log| options[:log] = log }
   opt.on('-o', '--on',              'turn on node')                            { options[:on] = true }
+  opt.on('-p', '--sol-info',        'get Serial Over LAN (SOL) info')          { options[:get_sol_info] = true }
```

Comment:
Tiny nitpick: i think that in all other cases we're using the long parameter name as the symbol name in the options hash. For instance, `--on` => `options[:on]`, `--kick` => `options[:kick]` etc. Might want to keep with that convention.

Not going to block you on that though. My LGTM still stands.